lib/imageDisplay.ml

```diff
@@ -122,12 +131,10 @@ let print_portrait conf base p =
     [Rem] : Ne pas utiliser en dehors de ce module.                           *)
```

Comment:
Are you sure it is okay to expose this function?

Reply:
I dont see why not!!